ci-operator/config/openshift-kni/eco-ci-cd/openshift-kni-eco-ci-cd-main__cnf-network-4.17.yaml

```diff
@@ -0,0 +1,28 @@
+build_root:
+  image_stream_tag:
+    name: eco-ci-cd
+    namespace: telcov10n-ci
+    tag: eco-ci-cd
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: cnf-network-functional-tests
+  cluster: build05
+  restrict_network_access: false
+  steps:
```

Comment:
What is the trigger for this test? Do we need cron here?

Reply:
I don't want to have this job running yet. This is just initial PR, we'll add trigger when e2e pipeline is ready.